CHANGES

```diff
@@ -34,6 +34,10 @@
      list of built in objects, i.e. OIDs with names.
      [Richard Levitte]
 
+  *) Added support for Linux Kernel TLS data-path. The Linux Kernel data-path
+     improves application performance by removing data copies and providing
+     applications with zero-copy system calls such as sendfile and splice.
```

Comment:
Nit: Please add your name after this entry, as per the other entries in this file

Reply:
Done